adabot/arduino_libraries.py

```diff
@@ -80,6 +80,16 @@ def is_arduino_library(repo):
     else:
         return False
 
+def print_list_output(title="", list=[]):
```

Comment:
A few things here:

- I would rename the `list` argument. Using names that are in the standard library or CPython keywords can be troublesome.

- Couldn't `title` & `list` just be positional arguments, instead of keyword arguments? There is no check for existence in the code (they're assumed to be there), so having them as positional forces them to be passed in.

- Also, empty lists are frowned upon as default keyword arguments. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments